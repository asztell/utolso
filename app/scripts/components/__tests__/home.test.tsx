import React from "react";
import { render } from "@testing-library/react";

import { Home } from "../home";

describe("<Home />", () => {
  test("render Home", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
