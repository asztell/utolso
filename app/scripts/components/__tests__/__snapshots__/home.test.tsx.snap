// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> render Home 1`] = `
<div>
  <section
    class="PageContent"
  >
    Some page content
  </section>
</div>
`;
